File: 04-CSHARP/08-Funciones/01funciones.cs
// Funciones en C#

Console.WriteLine("Ingresa el nombre a mostrar");
var mensajeArg = Console.ReadLine();
Saludar(mensajeArg);

// Definir el procedimiento (No regresa ningun tipo de informacion)
void Saludar(string mensaje)
{
    Console.WriteLine($"Hola {mensajeArg}");
}

File: 02_C/08_funciones/01funciones.c
// Funciones en C
#include <stdio.h>

// Definir funcion (Procedimiento porque no regresa ningpun valor) saludar.
void saludar(char mensaje[])
{
    printf("%s\n", mensaje);
}

int main()
{
    char argMensaje[50];
    printf("Escribe un mensaje: ");
    scanf("%[^'\n']s", argMensaje);

    saludar(argMensaje);
    saludar(argMensaje);

    return 0;
}
